### PR TITLE
Docs: Add Gateway Plugin Support

### PR DESCRIPTION
**Pull request by Gidi233**
**What type of PR is this?**
/kind documentation

**Which issue(s) this PR fixes**:
a part of #651

### CODE REVIEW

File: docs/content/en/docs/fleet-manager/rollout/abtest.md [hunk 1]
```diff
@@ -331,6 +347,105 @@ If the status shows as `Initialized`, it means the initialization of rollout pro
 
 **Notes**: In the above configuration, we set the `kustomization.interval` to 0s. This disables Fluxcd's periodic synchronization of configurations between the local mirror and cluster. The reason is that Flagger needs to modify the replica counts in Deployments to complete its initialization process. If you are uncertain whether the replicas for all applications in your deployments are set to zero, it is recommended to also set `kustomization.interval` to 0s.
 
+### Using Nginx
```
Comment (LiZhenCheng9527) [inline docs/content/en/docs/fleet-manager/rollout/abtest.md:350]: I don't think it should be added in the same usage document. It should add another three level catalogue
`Abtest`, `Blue-Green Deploy` and `Canary`.
Documentation according to different ingresses in three directories

**Comment by LiZhenCheng9527** [comment]
I have submitted a PR #693. After the PR has been merged in, you split the content of the document and write the document in accordance with different ingress.

File: docs/content/en/docs/fleet-manager/rollout/canary/kuma-canary.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+        fleet: quickstart
+      kustomization:
+        targetNamespace: default
+        interval: 5m0s
+        path: ./kustomize
+        prune: true
+        timeout: 2m0s
+```
+
+**Notes:**There is a problem with the metric provided by the current flagger, so `customMetric` is used.Here is the detailed [API](/docs/references/app-api/#apps.kurator.dev/v1alpha1.Metric).
```
Comment (LiZhenCheng9527) [inline docs/content/en/docs/fleet-manager/rollout/canary/kuma-canary.md:186]:
```suggestion
**Notes:** There is a problem with the metric provided by the current flagger, so `customMetric` is used.Here is the detailed [API](/docs/references/app-api/#apps.kurator.dev/v1alpha1.Metric).
```

A space is needed here to validate the syntax. Please check this in this pr.

**Comment by LiZhenCheng9527** [comment]
@hzxuzhonghu PTAL

**Review by hzxuzhonghu — APPROVED** [review]

**Pull request merged**